verification_token/migrations/0004_migration.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.15 on 2020-11-12 13:38
+
+from django.db import migrations, models
+from django.utils.timezone import timedelta
+
+
+def recompute_value_for_new_field(apps, schema_editor):
+    VerificationToken = apps.get_model('verification_token', 'VerificationToken')
+    expiring_tokens_qs = VerificationToken.objects.filter(expiration_in_minutes__isnull=False)
+    expiration_minutes = set(expiring_tokens_qs.values_list('expiration_in_minutes', flat=True))
```

Comment:
better is use order_by and distinct

Reply:
OK, but will still need the resulting set of plain values - but it will definitely make it easier for the `set`.

Reply:
Come to think of it, how much will that help? If I use `distinct`, I will still get all the records (unless I use Postgres specific functions, which I tried to avoid)...

Reply:
This will help a lot. No disctinct is not postgres specific if you use it correctly

Reply:
A misunderstanding - of course `distinct` is not Postgres specific, but there's a version of that command that is Postgres specific. What I referred to was https://docs.djangoproject.com/en/2.2/ref/models/querysets/#distinct:

> On PostgreSQL only, you can pass positional arguments (*fields) in order to specify the names of fields to which the DISTINCT should apply. This translates to a SELECT DISTINCT ON SQL query. Here’s the difference. For a normal distinct() call, the database compares each field in each row when determining which rows are distinct. For a distinct() call with specified field names, the database will only compare the specified field names.